packages/ignite-cli/src/commands/sourceMap.js

```diff
@@ -0,0 +1,75 @@
+// @cliDescription  Digests the source map file to display proper feedback
+// @cliAlias source-map
+// ----------------------------------------------------------------------------
+
+const SourceMapConsumer = require('source-map').SourceMapConsumer
+const exitCodes = require('../lib/exitCodes')
+
+const printSourceResult = (smc, position, colors) => {
+  const {source, line, column, name} = smc.originalPositionFor(position)
+  const styledName = colors.red(name)
+  const styledSource = colors.yellow(source)
+
+  console.log(`${styledName}: ${styledSource}
```

Comment:
I'd recommend using `context.print.info`.  This means you'll have to put this function in the main `module.exports` function.